setup.cfg

```diff
@@ -1,4 +1,5 @@
 [pycodestyle]
+ignore = W602
```

Comment:
Could also add `# noqa` to the offending line in the functional test to ignore all errors for that one line for pycodestyle (yet pylint would still fail on it)